app/src/main/java/com/example/testapp/ui/MainActivity.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.testapp.ui
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.example.testapp.R
+
+class MainActivity : AppCompatActivity() {
```

Comment:
Rename this MainActivity to something meaningful as mentioned in the previous comments. Likse launchActivity, DashboardActivity etc

Reply:
renamed